lib/src/components/NxCombobox/__tests__/NxCombobox.test.tsx

```diff
@@ -0,0 +1,522 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { render } from '@testing-library/react';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+
+import userEvent from '@testing-library/user-event';
+
+import NxCombobox, { Props } from '../NxCombobox';
+
+describe('NxCombobox', function() {
+  const minimalProps: Props<string | number> = {
+        value: '',
+        onChange: () => {},
+        onSearch: () => {},
+        matches: []
+      },
+      quickRender = rtlRender(NxCombobox, minimalProps),
+      renderEl = rtlRenderElement(NxCombobox, minimalProps);
+
+  it('sets specified classNames and attributes on the top-level element', function() {
+    const component = renderEl({ className: 'foo', title: 'bar' });
+
+    expect(component).toHaveClass('foo');
```

Comment:
Check that it also maintains its built-in classes when a custom `className` is set.  See https://github.com/sonatype/sonatype-react-shared-components/blob/main/lib/src/components/NxToast/__tests__/NxToast.test.tsx#L45 for an example

Reply:
Thanks for the example, updated here 6dbc601488edec3acdf06ca1c289688de300a1a8.